Update additional BN functions (#5083)
ppy/osu-wiki: wiki/People/The_Team/Beatmap_Nominators/General_Information/en.md

# General information for Beatmap Nominators

> Are you a [Beatmap Nominator](/wiki/People/Beatmap_Nominators) *(BN)*, or interested in becoming one? If so, you have come to the right place! This article has all of the general information you will need to know as a new Beatmap Nominator.

Reviewer: ```suggestion
Are you a [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators) *(BN)*, or interested in becoming one? If so, you have come to the right place! This article has all of the general information you will need to know as a new Beatmap Nominator.
```

## Important links

> - [Beatmap Nominator Rules](/wiki/People/Beatmap_Nominators/Rules): Outlines both how to undertake various actions, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) *(NAT)* if you are unsure about something, and they will be happy to help.

Reviewer: ```suggestion
- [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules): Outlines both how to undertake various actions, as well as how not to do things - you would want to avoid breaking these in any situation, so be sure to read them carefully! You can always ask a member of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) *(NAT)* if you are unsure about something, and they will be happy to help.
```

> - [Code of Conduct for Mapping and Modding](/wiki/Ranking_Criteria/Code_of_Conduct): As a BN, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled.

Reviewer: ```suggestion
- [Code of Conduct for Mapping and Modding](/wiki/vi/Rules/Code_of_Conduct_for_Modding_and_Mapping): As a BN, you are expected to be an exemplary figure of the community, someone others look up to and respect, so misconduct may get you reprimanded or even expelled.
```

- [Ranking Criteria](/wiki/Ranking_Criteria): If you are already a BN, then you should be familiar with this. Essentially, it serves to explain what aspects of mapping are and are not allowed in the Ranked section, as well as including some general guidelines that people should try to follow in most cases. 

> - [Beatmap Veto Information](/wiki/People/Beatmap_Nominators/Beatmap_Veto): For beatmaps with subjective quality concerns that don't necessarily break the rules and guidelines of the Ranking Criteria, but which you feel which make the beatmap in question unfit for the Ranked section, the beatmap veto allows you to withhold a beatmap from Qualification.

Reviewer: ```suggestion
- [Beatmap Veto Information](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto): For beatmaps with subjective quality concerns that don't necessarily break the rules and guidelines of the Ranking Criteria, but which you feel which make the beatmap in question unfit for the Ranked section, the beatmap veto allows you to withhold a beatmap from Qualification.
```

![disqualify and reset buttons](img/reset_dq.png)

> **This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full BN. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.

Reviewer: ```suggestion
**This allows you to reset a beatmap's progress in the ranking proccess.** If a beatmap is currently nominated or qualified, you as a nominator are given the ability to reset the beatmaps' nomination(s) by posting an issue using a Problem stamp. If a beatmap is qualified, it can only be disqualified by a full BN. These options are usually used when there is a major or unrankable issue in the beatmap which should be addressed before considering it for ranking.
```

## Additional Beatmap Nominator functions

In addition to being able to nominate beatmaps or reset their nominations, BNs have other functions as well.

### Veto mediation

When a veto mediation is initiated, a randomly selected jury of BNs will be selected. If you are one of the selected members for a veto mediation, the NAT will send you a message with additional information on how to vote using the [BN Website](http://bn.mappersguild.com). If you are unable to participate, inform a NAT member so that a different BN can be chosen for the jury in your place.

### Quality Assurance

> Full BN may do Quality Assurance by checking qualified beatmaps for issues at any time. The Quality Assurance board can be viewed [on the BN website](https://bn.mappersguild.com/qualityassurance).

Reviewer 2: Full BN -> Full BNs

### Content Review

Any BN can submit to vote and vote on media for content review via the BN website, contributing to the moderation of songs and images used in beatmaps on osu!. The full voting process and how their votes are counted can be read on the [content voting process article](https://osu.ppy.sh/wiki/en/Rules/Content_Voting_Process).

### Mock Evaluations

> Full BNs may be assigned at random to do mock evaluations on BN Applications alongside the NAT's real ones. They can choose to opt out of this at any time, and doing assigned evaluations is not required. Mock evaluations are primarily used as information for the NAT to evaluate future NAT candidates, though details presented in mock evaluations may be added to applicant feedback on occassion or serve as tiebreakers if the NAT can not come to a conclusion on their own.

Reviewer 2: Can not -> cannot

> 

Reviewer: I can't reach the text for the beatmap veto link above to suggest an edit, but it should be `[veto a beatmap](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto)` (on line 36)

## Additional Beatmap Nominator functions

In addition to being able to nominate beatmaps or reset their nominations, BNs have other functions as well.

### Veto mediation

When a veto mediation is initiated, a randomly selected jury of BNs will be selected. If you are one of the selected members for a veto mediation, the NAT will send you a message with additional information on how to vote using the [BN Website](http://bn.mappersguild.com). If you are unable to participate, inform a NAT member so that a different BN can be chosen for the jury in your place.

### Quality Assurance

Full BNs may do Quality Assurance by checking qualified beatmaps for issues at any time. The Quality Assurance board can be viewed [on the BN website](https://bn.mappersguild.com/qualityassurance).

### Content Review

> Any BN can contribute to the moderation of songs and images by either submitting content for review or voting on it. The full voting process and how their votes are counted can be read on the [content voting process article](/wiki/en/Rules/Content_Voting_Process).

Reviewer: ```suggestion
Any BN can contribute to the moderation of songs and images by either submitting content for review or voting on it. The full voting process and how their votes are counted can be read on the [content voting process article](/wiki/Rules/Content_Voting_Process).
```
